go.mod

```diff
@@ -1,103 +1,106 @@
 module github.com/nknorg/nkn/v2
 
-go 1.19
+go 1.20
+
+replace github.com/nknorg/nnet => ../nnet
```

Comment:
don't forget to remove this replace and upgrade nnet to the latest version that includes https://github.com/nknorg/nnet/pull/46 to be compatible with go 1.20

Reply:
done. thanks for the reminder.